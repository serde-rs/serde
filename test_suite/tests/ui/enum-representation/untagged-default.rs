use serde_derive::Deserialize;

#[derive(Deserialize)]
#[serde(untagged)]
enum E {
    V1 { f: u8 },
    #[serde(default)]
    V2 { f: u8 },
}

fn main() {}
