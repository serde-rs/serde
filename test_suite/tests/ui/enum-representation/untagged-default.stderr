error: #[serde(default)] can only be used on variants of internally tagged enums
 --> tests/ui/enum-representation/untagged-default.rs:7:5
  |
7 | /     #[serde(default)]
8 | |     V2 { f: u8 },
  | |________________^
